pyproject.toml

```diff
@@ -83,7 +83,7 @@ constraint-dependencies = [
 
 [build-system]
 requires = [
-    "setuptools==67.7.2",
+    "setuptools==67.8.0",
```

Comment:
This is updating setuptools, not pre-commit hook as in the PR title and body.  Why??